mmv1/third_party/terraform/website/docs/guides/google_project_service.html.markdown

```diff
@@ -0,0 +1,107 @@
+---
+page_title: "User Guide for google_project_service"
+description: |-
+  An advanced user guide for the google_project_service resource
+---
+
+# User Guide - google_project_service
+
+## Enabling Multiple Services in a Config
+
+Users may want to activate many services simultaneously within a single Terraform config. The `google_project_service.service` field only supports a single value, but Terraform itself provides list iteration with [for_each](https://developer.hashicorp.com/terraform/language/meta-arguments/for_each).
+A simple example:
+
+```
+variable "services" {
+  type = list(string)
+}
+
+resource "google_project_service" "services" {
+  for_each = toset(var.services)
+  project = "my-project"
+  service = each.value
+}
+```
+
+For a more robust example, Google recommends this [project_services module](https://github.com/terraform-google-modules/terraform-google-project-factory/tree/master/modules/project_services). This module aims to simplify the configuration needed for service enablement to [this example](https://github.com/terraform-google-modules/terraform-google-project-factory/tree/master/modules/project_services#example-usage).
+
+## Request Rate Errors
+
+The service management API called by the google_project_service resource has rate limits on the requesting user account if no quota project is set. When using the default (credentials) quota project or a billing_project users enabling many GCP services simultaneously may experience rate limit errors from the API. `google_project_service` supports batching multiple resources into smaller requests, see the `batching` [reference documentation](https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/provider_reference.html#batching).
```

Comment:
nit: Mind making a pass so that resource or field names are in `code text`?

Reply:
made another pass, couldn't find anything outside of headers or titles